### PR TITLE
S01E20-开源之夏学生专访-王嘉树

### LINKED ISSUE #212: 推荐嘉宾 - [王嘉树]（个人成长系列）
### 栏目

个人成长系列-学生篇

### 嘉宾名称

王嘉树

### GitHub

https://github.com/wjsvec/

### 公司/组织/岗位

_No response_

### Twitter

_No response_

### 博客地址

_No response_

### 推荐理由（着重考虑开源爱好者能收获什么）

王嘉树，北京邮电大学研二在读，曾参与开源之夏 [TDengine visual studio code](https://github.com/taosdata/vscode-tdengine) 项目并顺利结题，活跃在 TDengine 社区。目前的兴趣在开源项目方面，曾在科大讯飞、电信研究院等实习。

作为开源之夏学生群体中活跃的典型代表，展现出了突出的组织能力和实践能力，体现了开源在学生群体中日益高涨的活力。

### 推荐人

李梦 @LiMeng00 

### note

- [X] 建议嘉宾 [订阅《开源面对面》的邮件列表](mailto:osf2f+subscribe@googlegroups.com)（发送邮件被视为订阅），方便后续发送录制的会议邀请。
- [X] 可参考 [播客录音指南](https://www.bilibili.com/video/BV1kq4y1c7Nu) 准备好录音设备与环境。

### PR DESCRIPTION
**Pull request by LinuxSuRen**
See also the unpublished audio link [here](https://rkawrngf0e.feishu.cn/file/APZhbQ3hUoHWGExRDqCc81Kmnql).

cc @MikeyWei @wjsvec 

close #212

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
Modify the audio according to feedback from @MikeyWei . The new link is https://rkawrngf0e.feishu.cn/file/Rckfb2fQPoImtuxvJxXcjZjhnpg

**Review by stevending1st — APPROVED** [review]
LGTM

File: shownotes/S01E20-开源之夏学生专访-王嘉树.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## 主持人
+
+- [Mikey](https://github.com/MikeyWei) 开源爱好者、一个码字的
+- [Rick](https://github.com/linuxsuren)（开源爱好者、业余开源布道者）
+
+## 时间线
+
+* 02:38 接触开源的机缘是源自企业对开源项目的诉求
+* 08:23 落选 Nebula Graph
```
Comment (wey-gu) [inline shownotes/S01E20-开源之夏学生专访-王嘉树.md:34]: Sorry about this!!! 😭

**Review by wey-gu — APPROVED** [review]
❤

**Review by wey-gu — APPROVED** [review]
❤

File: shownotes/S01E20-开源之夏学生专访-王嘉树.md (hunk 1 above)
Comment (LinuxSuRen) [reply shownotes/S01E20-开源之夏学生专访-王嘉树.md:34]: Actually, this is also a valuable experience.

**Comment by wjsvec** [comment]
补充：之前参加第一次开源之夏的活动时的照片
0RJX5UC1R)9ZA8$0%6W1{2U

**Review by wjsvec — COMMENTED** [review]
lgtm

File: shownotes/S01E20-开源之夏学生专访-王嘉树.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* 11:56 想到自己参与的开源项目会有人在用就会很开心
+* 14:23 简述参与开源之夏的过程
+* 19:31 除了微信跟导师还有哪些交流渠道
+* 25:59 大厂写代码可能只是在拧螺丝
+* 33:50 参与开源之夏过程中投入的时间
+* 38:24 如何了解项目导师的相关信息
+* 55:10 如何思考开源项目需求的问题
+* 58:54 最后的几点建议
```
Comment (wjsvec) [inline shownotes/S01E20-开源之夏学生专访-王嘉树.md:44]: 59:03 最后给申请开源之夏的新人一点贴士

写成这样子怎么样

**Review by wjsvec — COMMENTED** [review]
LGTM

**Review by wey-gu — APPROVED** [review]

**Pull request merged**